src/app/components/ms-contents/ms-contents.component.html

```diff
@@ -0,0 +1,52 @@
+<div class="msContents">
+    <evt-ms-desc-section [label]="'msContents'"></evt-ms-desc-section>
+    <div>
+        <ng-container *ngIf="data?.structuredData; else elseBlock">
+            <div class="section">
+                <div class="row no-gutters" *ngIf="data?.summary">
+                    <div class="col-1">
+                        <evt-ms-desc-section [label]="'summary'" [inlineLabel]="true"></evt-ms-desc-section>
+                    </div>
+                    <div class="col-6 col-content">
```

Comment:
Same as comment before. Why do you use "col-6" and "col-content"? 
Anyway, this information will be contained in a panel, so you should use all the available space. Beware of smaller screens (there are overlaps)!